File: app/src/main/java/com/example/utente10/galileo/bean/Shop.java
package com.example.utente10.galileo.bean;

import io.realm.RealmList;
import io.realm.RealmModel;
import io.realm.annotations.RealmClass;

@RealmClass
public class Shop implements RealmModel {
    String name;
    String description;
    Beacon beacon;
    RealmList<String> photos;
}

File: app/src/main/java/com/example/utente10/galileo/bean/Shop.kt
package com.example.utente10.galileo.bean

import io.realm.RealmList
import io.realm.RealmModel
import io.realm.annotations.RealmClass

@RealmClass
data class Shop(
    val name: String,
    val description: String,
    val beacon: Beacon,
    val photos: RealmList<String>? = null) : RealmModel